### PR TITLE
Update deps and instructions for hapi v.16 scrub

### PR DESCRIPTION
**Pull request by zemccartney**

### CODE REVIEW

**Comment by zemccartney** [comment]
- boom kept at 5, not 6, b/c 6 uses version of hoek that breaks on node 4
- set node 8 to 8.7.0 in travis config b/c that's the last version where native http2 is not exposed by default

File: test/index.js [hunk 1]
```diff
@@ -486,7 +486,7 @@ describe('Underdog', () => {
                 srv.stop(done);
             });
 
-            request.on('push', () => next(new Error('Should not make it here')));
```
Comment (devinivy) [inline test/index.js]: Returning an error wont do anything!  You'd have to at least throw it.  I think that `next` was possibly supposed to be `done`.

File: test/index.js [hunk 2]
```diff
@@ -531,7 +531,7 @@ describe('Underdog', () => {
...
+            request.on('push', () => new Error('Should not make it here'));
```
Comment (devinivy) [inline test/index.js]: Ditto!

File: package.json [hunk 3]
```diff
@@ -34,15 +34,15 @@
     "items": "2.x.x"
   },
   "peerDependencies": {
-    "hapi": ">=10 <16"
+    "hapi": ">=10 <17"
```
Comment (devinivy) [inline package.json]: If this doesn't work yet on hapi v16, then no need for this change.

File: .travis.yml [hunk 4]
```diff
@@ -3,5 +3,6 @@ language: node_js
 node_js:
   - "4"
   - "6"
+  - "8.7.0"
```
Comment (devinivy) [inline .travis.yml]: Why this version exactly?

File: package.json (hunk 3 above)
Comment (zemccartney) [reply package.json]: oh bugger, extra sloppy. sorry about that, thanks for catching 🙏

File: .travis.yml (hunk 4 above)
Comment (zemccartney) [reply .travis.yml]: :trollface:  oh man, yea. took me a bit to peek through their changelog to sort that out. 🎺

File: test/index.js (hunk 1 above)
Comment (zemccartney) [reply test/index.js]: ay, gotcha. thanks for pointing that out, slopslopslop. fixing now!

File: test/index.js (hunk 2 above)
Comment (zemccartney) [reply test/index.js]: 🙏

**Pull request merged**